.travis.yml

```diff
@@ -21,8 +21,13 @@ jobs:
       - MAKE_TARGET=test
       - TEST_PROFILE=vim-profile-master.txt
     - name: Installed Vim with checks
+      language: python
+      python:
+        - "2.7"
+        - "3.4"
```

Comment:
Not sure this works as expected.
Should be only "python: 3.7", and then python2 would be installed also.